### PR TITLE
addSidecar logic updated from include toYaml

### PR DESCRIPTION
**Pull request by adapured**
1/ Upstream Include tpl for the AdditionalSidecar changed to toYAML.
2/Template moved to .Values
3/Cleaned the __helper file.

Test Results: [SplunkQry](https://splunk.or1.adobe.net/en-US/app/search/search?q=search%20index%3Dplat_app_preprod%20ops-zookeeper-operator-84d65c5f4f-hf7br&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=-2h%40h&latest=now&sid=1624057604.17638_720275B1-EF7E-4B39-88AB-22ADA5118883)

### CODE REVIEW

File: charts/zookeeper-operator/templates/operator.yaml [hunk 1]
```diff
@@ -44,14 +44,14 @@ spec:
...
-{{ toYaml .Values.additionalEnv | indent 8 }}
+{{ toYaml .Values.additionalEnv | nindent 8 }}
         {{- end }}
         {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         {{- end }}
       {{- if .Values.additionalSidecars }}
-{{ toYaml .Values.additionalSidecars | indent 6 }}
+{{toYaml .Values.additionalSidecars | nindent 6 }}
```
Comment (amuraru) [inline charts/zookeeper-operator/templates/operator.yaml:54]: Why removing the space between {{ and toYaml?
You are not consistent with same call above

File: charts/zookeeper-operator/templates/operator.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
       {{- if .Values.additionalVolumes }}
       volumes:
-{{ toYaml .Values.additionalVolumes | indent 6 }}
+{{toYaml .Values.additionalVolumes | nindent 8 }}
```
Comment (amuraru) [inline charts/zookeeper-operator/templates/operator.yaml:26]:
```suggestion
{{ toYaml .Values.additionalVolumes | indent 8 }}
```

Comment (amuraru) [reply charts/zookeeper-operator/templates/operator.yaml:26]: Be consistent with the existing format of the file

File: charts/zookeeper-operator/templates/operator.yaml (hunk 1 above)
Comment (adapured) [reply charts/zookeeper-operator/templates/operator.yaml:54]: This PR closed

File: charts/zookeeper-operator/templates/operator.yaml (hunk 2 above)
Comment (adapured) [reply charts/zookeeper-operator/templates/operator.yaml:26]: This PR closed

**Pull request closed without merging**